src/app/core/common-components/confirmation-dialog/confirmation-dialog.service.ts

```diff
@@ -42,18 +44,17 @@ export class ConfirmationDialogService {
     buttons: ConfirmationDialogButton[] = YesNoButtons,
     closeButton = true,
   ): Promise<boolean> {
-    return firstValueFrom(
-      this.dialog
-        .open(ConfirmationDialogComponent, {
-          data: {
-            title: title,
-            text: text,
-            buttons: buttons,
-            closeButton: closeButton,
-          },
-        })
-        .afterClosed(),
-    );
+    const dialogRef = this.ngZone.run(() => {
```

Comment:
no, can't explain it :sweat_smile: 

It needs to run _inside_ ngZone change detection, otherwise I got an empty dialog and the text only showed after I resized the window. No idea why this started happening now, however ...